lib/cocoapods/validator.rb

```diff
@@ -425,6 +427,8 @@ def add_app_project_import
       add_swift_version(app_target)
       add_xctest(app_target) if @installer.pod_targets.any? { |pt| pt.spec_consumers.any? { |c| c.frameworks.include?('XCTest') } }
       app_project.save
+      Xcodeproj::XCScheme.share_scheme(app_project.path, 'App')
+      # Xcodeproj::XCScheme.share_scheme(@installer.pods_project.path, pod_target.label)
```

Comment:
@segiddins adding this causes the tests to fail. I think `pods_project` needs to be stubbed but the tests are a bit complicated to figure out. Let me know if something quick comes up for this.